internal/config/testdata/pipelines/pipeline_multi_step.hcl

```diff
@@ -0,0 +1,42 @@
+project = "foo"
+
+pipeline "foo" {
+  step "zero" {
+    image_url = "example.com/test"
+
+    use "test" {
+      foo = "qubit"
+    }
+  }
+
+  step "one" {
+    image_url = "example.com/second"
+
+    use "test/exec" {
```

Comment:
Aha, so I looked into why this wasn't causing a failure. I use this config specifically for the HCL parsing part of `config/pipelines`, where the actual `use "label"` validation for steps really happens over in `PipelineProtos()`. I removed the namespacing since it doesn't make sense for the config test this was being used for.